app/models/crew_position.rb

```diff
@@ -0,0 +1,5 @@
+class CrewPosition < ApplicationRecord
+
```

Comment:
your crew associations depend on an association being written in `CrewPosition`.  `CrewPosition` needs to be connected to `Movie` with an association for `Crew` to get movie objects.  It also needs to be connected to crew.